File: 공예진/programmers/P181914.java
//문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181914
//시간: 5.19ms
//메모리: 75.3MB

class Solution {
    public int solution(String number) {
        int sum = 0;
        for (int i = 0; i < number.length(); i++) {
            sum += number.charAt(i) - '0';
        }

        return sum % 9;
    }
}

File: 강재연/programmers/P181914.js
// 문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181914
// 시간: 4.19 ms
// 메모리: 33.4 MB

function solution(number) {
    let answer = 0;
    // +n을 하면 문자 → 숫자 변환 가능
    for(const n of number) answer += +n;
    return answer % 9;
}
